orangecontrib/text/widgets/ownyt.py

```diff
@@ -10,14 +10,14 @@
 from orangecontrib.text.corpus import Corpus
 from orangecontrib.text.nyt import NYT, MIN_DATE
 from orangecontrib.text.widgets.utils import CheckListLayout, DatePickerInterval, QueryBox, \
-    gui_require, OWConcurrentWidget, asynchronous
+    gui_require, asynchronous
 
 
 class IO:
     CORPUS = "Corpus"
 
 
-class OWNYT(OWConcurrentWidget):
+class OWNYT(OWWidget):
```

Comment:
After stopping the search this widget occasionally does not send the data out (None on output). Not quite sure if this is NYT specific or a general problem. However, I did not notice this elsewhere.